opensrp-path-zeir/src/test/java/org/smartregister/path/fragment/ChildRegisterDataFragmentTest.java

```diff
@@ -127,6 +127,12 @@ public void testResetAdapterData() {
         Assert.assertEquals(fragment.getmAdapter().getItemCount(), detailMap.size());
     }
 
+    @Test
+    public void testAddUnFormattedNumberFields()
```

Comment:
This test naming does not follow convention. It is not clear to tell what this unit test is trying to achieve.

Please use the pattern `test[MethodName_StateUnderTest_ExpectedBehavior]`